fe/fe-core/src/main/java/org/apache/doris/catalog/Replica.java

```diff
@@ -239,12 +239,19 @@ public void setNeedFurtherRepair(boolean needFurtherRepair) {
         this.furtherRepairSetTime = System.currentTimeMillis();
     }
 
-    // only update data size and row num
+    // for compatibility
     public synchronized void updateStat(long dataSize, long rowNum) {
         this.dataSize = dataSize;
         this.rowCount = rowNum;
     }
 
+    // only update data_size, row_num, version_count
```

Comment:
I think this comment is useless. Because we can get these information from function parameters